esmvaltool/cmorizers/data/downloaders/downloader.py

```diff
@@ -0,0 +1,45 @@
+"""Downloader base class."""
+
+import os
+
+
+class BaseDownloader():
+    """Base class for all downloaders.
+
+    Parameters
+    ----------
+    config : dict
+        ESMValTool's user configuration
+    dataset : str
+        Dataset to download
+    overwrite : bool
+        Overwrite already downloaded files
+    """
+    def __init__(self, config, dataset, overwrite, tier=3):
```

Comment:
Related to my other comment on the wrong tiers. Instead of assuming `tier=3` by default, could the `tier` be read from the `dataset.yml` file automatically? This would reduce the number of times the tier should be included for each cmorizer (cmor_config file, python scripts, dataset.yml, doc,...).

Reply:
Fixed now